activities/value-and-impact/index.md

```diff
@@ -4,9 +4,14 @@ type: Index
 
 # Value and impact
 
-These are all the activities, documents and templates for value and impact:
+These are activities, documents and evidence for how we create value and impact:
+
+## Value
 
 * [User mapping and value proposition hypotheses](user-mapping/index.md) for potential users of Foundation for Public Code products and activities
 * [Existing pain points](pain-points/index.md) for users of the Foundation for Public Code products and activities
+
+## Impact
+
 * [How public code supports the Sustainable Development Goals](sustainable-development-goals.md)
```

Comment:
I also love this!

Reply:
can this be approved? Then I can merge :)